internal/util/importutil/parquet_parser.go

```diff
@@ -205,28 +211,29 @@ func verifyFieldSchema(dataType, elementType schemapb.DataType, fileField arrow.
 	case arrow.BOOL:
```

Comment:
Would it be preferable to convert arrow.XX to DataType and provide a function, IsConvertible(src, dst DataType) bool, instead?